src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -76,7 +77,7 @@ export interface TableProps {
   items: SavedObjectWithMetadata[];
   itemId: string | (() => string);
   totalItemCount: number;
-  onQueryChange: (query: any) => void;
```

Comment:
can this be made optional?

Reply:
The `filterFields` param? It can, but in practical use it will always at least contain `type`.

Reply:
FYI reverted this back to the original parseQuery, but modified the function to also extract namespaces.